src/pages/LendingTeam/LendingTeam.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+	<www-page>
+		<kv-page-container>
+			<kv-grid class="tw-grid-cols-6
+			lg:tw-grid-cols-12"
+			>
+				<div class="tw-col-span-12
+				md:tw-order-first
+				lg:tw-col-span-8"
+				>
+					<h1 class="tw-text-h1
+					tw-mt-4 tw-mb-2"
+					>
+						Lending Teams
+					</h1>
+					<h2 class="tw-text-h3
+					tw-mb-4"
+					>
+						Connect with Kiva's Lender Community
+					</h2>
+					<h3 class="tw-text-base
+					tw-mb-4"
+					>
+						Lending teams are self-organized groups built around common interests, school
+						affiliation or location. Join as many teams as you'd like, and rally around
+						shared lending goals by counting each of your loans at checkout toward the
+						team's impact. To get started, simply join an existing team or create a new
+						one and invite your friends!
+					</h3>
+					<div class="tw-flex-row
+					tw-gap-0.5
+					tw-mb-4"
+					>
+						<kv-button variant="primary"
+							to="/teams/create"
+							class="tw-w-full md:tw-w-auto"
+						>
+							Start Team
+						</kv-button>
+						<kv-button variant="secondary"
+							to="/teams/my-teams"
+							class="tw-w-full md:tw-w-auto "
+						>
+							My Teams
+						</kv-button>
+						<kv-button variant="secondary"
+							to="/teams/playbook"
+							class="tw-w-full md:tw-w-auto "
+						>
+							Team Playbook
+						</kv-button>
+						<kv-button variant="secondary"
+							to="/kiva-community-guidelines"
+							class="tw-w-full md:tw-w-auto "
+						>
+							Community Guidelines
+						</kv-button>
+					</div>
+				</div>
+				<div class="tw-col-span-12
+				md:tw-col-start-7
+				md:tw-col-span-6
+				md:tw-row-start-1-row-divide-1/2
+				lg:tw-col-span-4
+				lg:tw-col-start-9
+				lg:tw-row-span-2"
+				>
+					<div class="tw-text-h4 tw-mt-4">
+						TEAM LEADERBOARDS
+					</div>
+					<team-leaderboards board-type="overallLoanedAmount" />
+					<team-leaderboards board-type="memberCount" />
+				</div>
+				<div class="tw-col-span-12
+				md:tw-col-span-6
+				md:tw-row-start-1-row-divide-1/2
+				lg:tw-col-span-8"
+				>
+					<team-listing />
+				</div>
+			</kv-grid>
+		</kv-page-container>
+	</www-page>
+</template>
+
+<script>
+import WwwPage from '@/components/WwwFrame/WwwPage';
+import KvPageContainer from '~/@kiva/kv-components/vue/KvPageContainer';
+import KvButton from '~/@kiva/kv-components/vue/KvButton';
+import KvGrid from '~/@kiva/kv-components/vue/KvGrid';
+import TeamListing from './TeamListing';
+import TeamLeaderboards from './TeamLeaderboards';
+
+export default {
+	name: 'LendingTeam',
+	components: {
+		WwwPage,
+		KvPageContainer,
+		KvButton,
+		KvGrid,
+		TeamListing,
+		TeamLeaderboards
+	},
+	methods: {
+		handleSearchQuery(queryString) {
+			this.queryString = queryString;
+			console.log(this.queryString);
```

Comment:
Is this method for future functionality? I'm not seeing `queryString` used in this component.

Reply:
The variable is watched and handled below.